internal/endpoint/http.go

```diff
@@ -57,8 +57,10 @@ func (conn *HTTPConn) Send(msg string) error {
 	if _, err := io.Copy(ioutil.Discard, resp.Body); err != nil {
 		return err
 	}
-	// we only care about the 200 response
-	if resp.StatusCode != 200 {
+	// Only allow responses with status code 200, 201, and 202
+	if resp.StatusCode != http.StatusOK &&
```

Comment:
Should this be a `or` not an `and`?

Reply:
Nah, If they were `or`'s the statment would always be true